src/napari_imagej/_module_utils.py

```diff
@@ -71,6 +90,18 @@ def python_type_of(module_item: "jc.ModuleItem"):
     )
 
 
+@module_item_converter(priority=Priority.HIGH)
+def placeholder_converter(item: "jc.ModuleItem"):
+    """
+    Checks to see if this type can be satisfied by a placeholder.
+    For a placeholder to work, it MUST be a pure input.
+    This is because the python type has no functionality, as it is just an Enum choice.
+    """
+    if item.isInput() and not item.isOutput():
+        java_type = item.getType()
+        return type_placeholders().get(java_type, None)
```

Comment:
We should test `python_type_of` with a `ItemIO.BOTH ` and/or `ItemIO.OUTPUT` with an enum-like to ensure that it only works for pure inputs.

Reply:
Done!